moviesurfer/cli/main.py

```diff
@@ -1,2 +1,19 @@
+import os
+import wget
+import zipfile
+
+# Function to download data
+def download_data():
+    output_dir = './moviesurfer/data'
+    url = 'http://files.grouplens.org/datasets/movielens/ml-25m.zip'
+    filename = wget.download(url, out=output_dir)
```

Comment:
if the file already exists, will it overwrite it? If it crashes when it exists, then it is ok. If it doesn't, we should throw an exception

Reply:
Agreed. It would be better if we adress that explicitly. Thanks a lot!